apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/RowFiltersTest.cs

```diff
@@ -0,0 +1,245 @@
+﻿// Copyright 2017 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Protobuf;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Google.Cloud.Bigtable.V2.Tests
+{
+    public class RowFiltersTest
+    {
+        [Fact]
+        public void BlockAllFilter()
+        {
+            var filter = RowFilters.BlockAllFilter();
+            Assert.True(filter.BlockAllFilter);
```

Comment:
I'm in two minds about this - it's not testing as much as `Assert.Equal(new RowFilter { BlockAllFilter = true }` but then that really would be a copy of the production code. Ditto a lot of the rest. This is the problem with testing mostly-trivial (but still useful, of course) code.

Reply:
Yeah, I agree. I figured rather than testing for a copy/pasted creation of what the implementation does, I'd rather test for the effect of those instances being created. Like you said, it's trivial and really just a smoke test to catch something that went terribly wrong.